QStore4S/pom.xml

```diff
@@ -200,8 +220,8 @@
 			<!-- Dev Env. Properties -->
 			<properties>
 				<!-- Neo4j Properties -->
-				<neo4j.user>${neo4j.username}</neo4j.user>
-				<neo4j.pw>${neo4j.password}</neo4j.pw>
+				<neo4j.user>neo4j</neo4j.user>
+				<neo4j.pw>password</neo4j.pw>
```

Comment:
These need to be variables like the other parameters (e.g. deploy.path). Right now you still hardcode username and password (username = neo4j, password = password).